### PR TITLE
Allowing SecGen to be run on servers/droplets

### PR DESCRIPTION
**Pull request by egg82**

### CODE REVIEW

**Comment by Jjk422** [comment]
Thank you @egg82 for the pull request.
After testing, the pull request seems to be working as desired.
However, this will not automatically spawn the vms if on a gui.
Spawning vms on a gui can be changed via adding a gui option, I have created a quick test branch to show how this can be done, the branch can be found at 'https://github.com/jjk422/SecGen/tree/new_added_gui/server_option_follows_from_pr/60'.
The branch is not entirely optimized and passing a hash of options may be better then a singular variable.
In summary, I do not mind merging your branch into master as it is, however if you want to change/modify the branch into allowing a --gui-output option to spawn the machines I can put off the merge until then.

**Comment by egg82** [comment]
Your implementation is far better than mine, of course. Allowing options for GUI vs Non-GUI is always good. Updated the branch to reflect the new changes. Only minor change I made was to force-set "vb.gui = false" in the "else" section of lib/templates/Vagrantfile.erb - overriding any defaults that may be set by the master instead of assuming what defaults are set.

**Comment by Jjk422** [comment]
Thanks for the quick response.
Tested and seems to be working perfectly so merging with main branch.
